samples/embedded-auth-with-sdk/src/main/resources/templates/verify.html

```diff
@@ -69,9 +69,44 @@ <h2 class="text-center">Verify Code</h2>
             <button id="verify-btn" type="submit" class="btn btn-primary btn-block">Submit</button>
         </div>
     </form>
+
+    <form name="pollRedirectForm" th:action="@{/poll}" method="get" hidden>
+        <noscript>
+            <input type="submit"/>
+        </noscript>
+    </form>
 </div>
 
 <div th:replace="fragments/footer :: footer"></div>
 
+<script th:inline="javascript">
+    function sleep(ms) {
+        return new Promise(resolve => setTimeout(resolve, ms));
+    }
+
+    function poll() {
+        fetch("/poll", {method: 'GET'})
+            .then(response => {
+                response.body.getReader().read()
+                    .then((body) => {
+                        if (body.value !== undefined) {
+                            document.pollRedirectForm.submit()
+                        }
+                    })
+            }).catch(function (err) {
+                console.error(err);
+            }
+        );
+    }
+
+    async function sendPollRequest() {
+        do {
+            poll();
+            await sleep(1000);
```

Comment:
The response from the server contains the expected backoff duration. We should use that instead of hard coding.

Reply:
Thank you!
I've updated this part of the code.